### PR TITLE
Autodetect content directory for capture cleanup tool

### PR DESCRIPTION
**Pull request by sftim**
Change `scripts/replace-capture.sh` to work out `$K8S_WEBSITE` if you didn't set it.

### CODE REVIEW

File: scripts/replace-capture.sh [hunk 1]
```diff
@@ -1,8 +1,37 @@
...
+
+find_content_dir() {
+  local self
+  local top
+  if command git rev-parse --is-inside-work-tree > /dev/null 2>&1 ; then
+    self="$0"
+    top="$(command git rev-parse --show-toplevel)"
+    while ( cd "${top}/.." && command git rev-parse --is-inside-work-tree> /dev/null 2>&1 ); do
+      top="$( cd "${top}/.." && "${self}" )"
+    done
```
Comment (kbhawkey) [inline scripts/replace-capture.sh:17]: nit: This seems a bit excessive with the goal of setting a single variable. Also, I am not convinced this
script should remain as a docs script. This was written as a one off task script.

Comment (sftim) [reply scripts/replace-capture.sh:17]: It's copied from some other code I had which includes robust error handling.

Comment (kbhawkey) [reply scripts/replace-capture.sh:17]: I read through the changes but have not rerun the script to validate the CONTENT_DIR setting.
/lgtm

**Pull request merged**